vault/resource_secrets_sync_gcp_destination.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	syncutil "github.com/hashicorp/terraform-provider-vault/internal/sync"
+)
+
+const (
+	gcpSyncType = "gcp-sm"
+)
+
+var gcpSyncWriteFields = []string{
+	consts.FieldCredentials,
+	consts.FieldSecretNameTemplate,
+	consts.FieldCustomTags,
+}
+
+var gcpSyncUpdateFields = []string{
+	consts.FieldCredentials,
+	// consts.FieldSecretNameTemplate,
+	// consts.FieldCustomTags,
```

Comment:
intentional?

Reply:
Yup! Wanted to make sure they were editable, thanks for confirming!